ports/sqlitecpp/portfile.cmake

```
@@ -25,6 +25,7 @@ vcpkg_configure_cmake(
         -DSQLITECPP_INTERNAL_SQLITE=OFF
         -DSQLITE_ENABLE_COLUMN_METADATA=OFF
         -DSQLITECPP_INTERNAL_SQLITE=OFF
+        -DSQLITECPP_USE_STACK_PROTECTION=OFF
```

```suggestion
        ${USE_STACK_PROTECTION}
```
If the link error only occurs with mingw64, Please set a usage condition for this option before vcpkg_configure_cmake.
e.g.
set(USE_STACK_PROTECTION "-DSQLITECPP_USE_STACK_PROTECTION=ON") #(I'm not sure if this option is ON in other triplets, please help confirm)
if(VCPKG_TARGET_IS_MINGW)
    set(USE_STACK_PROTECTION "-DSQLITECPP_USE_STACK_PROTECTION=OFF")
endif()